README.md

```diff
@@ -1,24 +1,197 @@
-# README
+<!-- TABLE OF CONTENTS -->
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [🚀 Live Demo](#live-demo)
+    - [Live video](#live-video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
 
-* System dependencies
+# 📖 EXPENSES TRACKER APP <a name="about-project"></a>
 
-* Configuration
+> Expenses-tracker-app is an application that allows the user to:
 
-* Database creation
+> - register and log in, so that the data is private to them.
+> - introduce new transactions associated with a category.
+> - see the money spent on each category.
 
-* Database initialization
+It is basically a mobile web application where you can manage your budget:
+you have a list of transactions associated with a certain category,
+so that you can see how much money you spent and on what.
 
-* How to run the test suite
+**
+expenses-tracker-app** is a Ruby on Rails application...
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Client / Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Deploy</summary>
+  <ul>
+    <li><a href="https://render.com/">Render</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database </summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Tests</summary>
+  <ul>
+    <li><a href="https://rspec.info/">Rspec</a></li>
+    <li><a href="https://github.com/teamcapybara/capybara">Capibara</a></li>
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Hi @basitali111, this project really needs to be deployed and accessible online all over the world by users. It's a requirement of this project.  Kindly check out this article on [Zendesk ](https://microverse.zendesk.com/hc/en-us/articles/10883335713043), a new way of deploying projects at Microverse. This was an [announcement](https://microverse.zendesk.com/hc/en-us/articles/10490872573715) that was made some weeks ago.

Reply:
@FrederickMih  i try that article it is also become paying now

Reply:
Hi @basitali111, render is still  free, Kindly check out this article on [Zendesk ](https://microverse.zendesk.com/hc/en-us/articles/10883335713043) on deploying applications on render, It would be better to deploy your application  so users don't have to clone your repository to see your app

Reply:
@prolajumokeoni i try it in my side when i click on it show me credit card and one more think it take time and its last hour try to understand my situation

Reply:
- There is the option to choose a free plan, Kindly go through the article on [Zendesk ](https://microverse.zendesk.com/hc/en-us/articles/10883335713043) carefully